xmodule/docs/decisions/0005-send-data-to-edx-submission.rst

```diff
@@ -0,0 +1,191 @@
+#########################################################
+Implementation to send student response to edx-submission
+#########################################################
+
+Status
+******
+
+Accepted.
+
+2025-03-06
+
+Context
+*******
+
+Traditionally, the Open edX platform has utilized the `XQueue` service to handle student assignment submissions. However, to leverage a more modern and efficient architecture, there is a need to transition these submissions to the `edx-submission` service. This transition aims to improve the scalability and maintainability of the submission handling process.
+
+Decision
+********
+
+The following changes have been implemented to redirect student submissions to the `edx-submission` service:
+
+1. **Introduction of `send_to_submission` Function in `xqueue_submission.py`:**
+   - **Functionality:** A new function, `send_to_submission`, has been developed to process student responses. It parses essential information such as `course_id`, `item_id`, `student_id`, and `max_score` from the submission data.
```

Comment:
Please add blank lines above and below each bullet. Check the rendered file - you can see currently this renders as a giant block instead of a bulleted list. https://github.com/openedx/edx-platform/blob/ef018510eca65c47d26a4a60975b518820f04634/xmodule/docs/decisions/0005-send-data-to-edx-submission.rst